config/urls.py

```diff
@@ -27,112 +27,186 @@
-    path('recruitment/', include('recruitment.urls')),
-
+    path("autoemails/", include("autoemails.urls")),
+    path("api/v1/", include("api.urls")),  # REST API v1
+    path("dashboard/", include("dashboard.urls")),
+    path("requests/", include("extrequests.urls")),
+    path(
+        "forms/", include("extforms.urls")
+    ),  # external, anonymous user-accessible forms
+    path("fiscal/", include("fiscal.urls")),
+    path("reports/", include("reports.urls")),
+    path("trainings/", include("trainings.urls")),
+    path("workshops/", include("workshops.urls")),
+    path("select_lookups/", include("workshops.lookups")),  # autocomplete lookups
+    path("terms/", include("consents.urls")),
+    path("communityroles/", include("communityroles.urls")),
+    path("recruitment/", include("recruitment.urls")),
     # for webhooks from Mailgun
     # path('mail_hooks/', include('anymail.urls')),
-
     # django views for authentication
-    path('account/login/',
-         auth_views.LoginView.as_view(
-             template_name="account/login.html",
-             extra_context={"title": "Log in"},
-         ),
-         name='login'),
-
-    path('account/logout/',
-         logout_then_login_with_msg,
-         name='logout'),
-
-    path('account/password_reset/',
-         auth_views.PasswordResetView.as_view(
-             template_name="account/password_reset.html",
-         ),
-         name='password_reset'),
-
-    path('account/password_reset/done/',
-         auth_views.PasswordResetDoneView.as_view(
-             template_name="account/password_reset_done.html",
-         ),
-         name='password_reset_done'),
-
-    re_path(r'^account/reset/(?P<uidb64>[0-9A-Za-z_\-]+)/'
-            r'(?P<token>[0-9A-Za-z]{1,13}-[0-9A-Za-z]{1,20})/$',
-            auth_views.PasswordResetConfirmView.as_view(
-                template_name="account/password_reset_confirm.html",
-            ),
-            name='password_reset_confirm'),
-
-    path('account/reset/done/',
-         auth_views.PasswordResetCompleteView.as_view(
-             template_name="account/password_reset_complete.html",
-         ),
-         name='password_reset_complete'),
-
+    path(
+        "account/login/",
+        auth_views.LoginView.as_view(
+            template_name="account/login.html",
+            extra_context={"title": "Log in"},
+        ),
+        name="login",
+    ),
+    path("account/logout/", logout_then_login_with_msg, name="logout"),
+    path(
+        "account/password_reset/",
+        auth_views.PasswordResetView.as_view(
+            template_name="account/password_reset.html",
+        ),
+        name="password_reset",
+    ),
+    path(
+        "account/password_reset/done/",
+        auth_views.PasswordResetDoneView.as_view(
+            template_name="account/password_reset_done.html",
+        ),
+        name="password_reset_done",
+    ),
+    path(
+        "account/reset/<uidb64>/<token>/",
```

Comment:
The whole file has been re-formatted with Black, but this is the only path that I actually changed.